internal/rbd/nodeserver.go

```diff
@@ -563,6 +602,19 @@ func (ns *NodeServer) createStageMountPoint(ctx context.Context, mountPath strin
 	return nil
 }
 
+func getDeviceSize(ctx context.Context, devPath string) (int64, error) {
+	mp := volume.NewMetricsBlock(devPath)
```

Comment:
okay, this does not work as a check for resizing. because if the rbd image is 8Gib when we check the size of the device it always returns the size of the device (8Gib), not the filesystem size which we can use for comparison.

@nixpanic @humblec @pkalever any other idea which we can use for size comparison to doing resize? or another option is to blindly resize which is resize the filesystem if it's required or else it will return success.

Reply:
Discard this one. The latest changes should fix it.